turbo.json

```diff
@@ -3,11 +3,22 @@
   "pipeline": {
     "build": {
       "cache": false,
-      "dependsOn": ["^build:shared", "^build:typography", "build:tailwind-config", "build:release",  "build:next", "build:nuxt", "build:docs"],
-      "outputs": ["dist/**"]
+      "dependsOn": [
```

Comment:
locally configured autoformatters <3

Reply:
Btw. we could autoformat this file with something. Prettier?

Reply:
after this change I did added prettier ext in vsc because of this change, but I though its good change anyway so I left, but if you feel I can revert it as it was previously